### ISSUE TITLE
"Display more" button in quote / code boxes is not accessible via keyboard navigation

### ISSUE THREAD

**Issue by BurntimeX** [WoltLab/WCF#3702]
ref https://community.woltlab.com/thread/286250-accessibility-funktionen-%C3%BCber-tastatur-nicht-erreichbar/
